.github/workflows/ci.yml

```diff
@@ -26,7 +26,13 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
-    - uses: actions/setup-dotnet@v3.0.3
+    - name: Install .NET 3.1 SDK
```

Comment:
We can get rid of `netcoreapp3.1` since it's no longer supported: https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core

Reply:
I was going to do that but there seems to be some friction to work through first: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/792#discussion_r1030156435

My goal here is just to get stuff stable again and then we can tackle that? 🤷